_sass/pages/_home.scss

```diff
@@ -0,0 +1,160 @@
+      margin: 1em auto 0.75em auto;
+    }
+
+    .subtitle {
+      max-width: 960px;
+      margin: auto;
+      color: $purple-500;
+    }
+
+    .logo {
+      max-height: 100px;
+    }
+  }
+
+  .counters {
+    display: flex;
+    justify-content: space-around;
+    margin: 3em 0;
+  }
+
+  .counter {
+    background: #fff;
+    box-shadow: 0 0 0 1px $purple-100, 0 0 0 10px #FFF, 0 0 0 11px $purple-100;
+    width: 30%;
+    max-width: 300px;
+
+    &__header { flex: 1; }
+
+    &__footer {
+      flex: 2;
+      color: $purple-500;
+      padding: 0 1em;
+      line-height: 1;
+
+      strong {
+        color: inherit;
+        font-size: 1.6em;
+        display: block;
+      }
+    }
+
+    &__content {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      height: 100%;
+      text-align: center;
+      font-variant: all-small-caps;
+    }
+
+    &__number {
+      flex: 3;
+      font-size: 6em;
+      line-height: 1;
+      font-variant: normal;
+      font-weight: 600;
+      color: $blue-500;
+      padding-top: 0.06em;
+      background: linear-gradient(45deg, #3e77be 0%, #3e77be 12.5%, #5a91cd 12.5%,  #5a91cd 25%,#6da3d7 25%, #6da3d7 37.5%,#99bfe3 37.5%, #99bfe3 50%,#659bd3 50%, #659bd3 50.5%,#99bfe3 50.5%, #99bfe3 53%,#659bd3 53%, #659bd3 53.5%,#99bfe3 53.5%, #99bfe3 75%,#83b1dd 75%, #83b1dd 100%),linear-gradient(135deg, #7767a6,#271763);
+      -webkit-background-clip: text;
+      -webkit-text-fill-color: transparent;
+    }
+  }
+
+  .button-in-text { padding-top: 2em; }
+
+  .media-header {
+    margin-bottom: 1em;
+
+    .title { font-weight: bold; }
+  }
+
+  .date {
+    flex-direction: row;
+    display: flex;
+    min-height: 2.5rem;
+    flex: 1 1 2rem;
+
+    &__day {
+      flex: 0;
+      text-transform: capitalize;
+      font-size: 2rem;
+      line-height: 2.5rem;
+      margin-right: .5rem;
+      color: $purple-400;
+      font-weight: 300;
+      white-space: nowrap;
+    }
+
+    &__date {
+      font-size: .85rem;
+      line-height: .85rem;
+      justify-content: center;
+      display: flex;
+      flex-direction: column;
+      font-weight: 300;
+    }
+
+    &__month {
+      color: #4a4a4a;
+      text-transform: capitalize;
+      font-size: .85rem;
+      line-height: 1.2;
+      justify-content: center;
+      display: flex;
+      flex-direction: column;
+      font-weight: 300;
+    }
+
+    &__weekday {
+      font-size: .75em;
+      color: $blue-700;
+      text-transform: capitalize;
+      text-align: left;
+    }
+  }
+
+  .event {
+    .location {
+      display: flex;
+      align-items: center;
+
+      &__icon {
+        font-size: 1.8em;
+        color: $purple-400;
+        margin-right: 0.1em;
+        height: 2.5rem;
+        width: 2.5rem;
+      }
+
+      &__text {
+        font-size: 0.85em;
+        line-height: 1.2;
+      }
+    }
+  }
+
+  .new-supporter {
+    .tags {
+      justify-content: center;
+    }
+
+    .tag {
+      &.is-state,
```

Comment:
Why is all this stuff that nested? I'd expect all `.tag.is-*` across the website to have the same color, and this way of declaring instructions means we'll have to duplicate it several times  😕